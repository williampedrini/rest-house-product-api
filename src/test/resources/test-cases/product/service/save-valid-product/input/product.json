{
  "id": null,
  "name": "Product 01",
  "quantity": 1
}