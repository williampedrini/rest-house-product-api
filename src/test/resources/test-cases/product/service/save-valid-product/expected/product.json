{
  "id": "5cbe47700000000000000000",
  "name": "Product 01",
  "quantity": 1
}